backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -64,7 +64,7 @@
     /**
      * The current session frame data
      */
-    private SessionFrame frame;
+    public SessionFrame frame;
```

Comment:
Can you revert this change and access `frame` via the existing `getFrame` method?